client/js/color.js

```diff
@@ -64,4 +66,38 @@ export function contrastAnyColor(inputColor, intensity, direction) {
   const toHexString = (r, g, b) => '#' + r.toString(16).padStart(2, '0') + g.toString(16).padStart(2, '0') + b.toString(16).padStart(2, '0');
   return toHexString(...adjustColor(r, g, b, modifier));
   return defaultColor;
+}
+
+export function calcHue(startingColors) {
```

Comment:
I don't like this function name. I would expect calcHue to take a color and return the hue value of that color.

Reply:
`randomHue`?